integreat_cms/cms/migrations/0084_auto_20231129_1607.py

```diff
@@ -0,0 +1,72 @@
+# Generated by Django 3.2.23 on 2023-11-29 16:07
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("cms", "0083_add_page_based_offer_fields"),
+    ]
```

Comment:
```suggestion
class Migration(migrations.Migration):
    """
    Change wording of "review" to "approval"
    """

    dependencies = [
        ("cms", "0083_add_page_based_offer_fields"),
    ]
```

Reply:
In addition, could you rename the file name to "0084_rename_review_to_approval.py" ?